Server/src/cli/commands/audio.py

```
@@ -0,0 +1,130 @@
+@click.option(
+    "--clip", "-c",
+    default=None,
+    help="Audio clip path to play."
+)
+@click.option(
+    "--search-method",
+    type=click.Choice(["by_name", "by_path", "by_id"]),
+    default=None,
+    help="How to find the target."
+)
+def play(target: str, clip: Optional[str], search_method: Optional[str]):
```

Ruff (0.14.10)

Unused function argument: `clip`

(ARG001)